ci/style.rs

```diff
@@ -140,9 +139,6 @@ fn check_style(file: &str, path: &Path, err: &mut Errors) {
         } else if line.starts_with("type ") && !in_impl {
             State::Typedefs
         } else if line.starts_with("s! {") {
-            s_macros += 1;
```

Comment:
I had to allow multiple `s!` macros in one module because the `s_no_extra_traits!` macro no longer exists.